### ISSUE TITLE
Potential conflict with Inactive Tokens

### ISSUE THREAD

**Issue by txm3278** [manuelVo/foundryvtt-drag-ruler#252]
This warning is logged when moving any token
![image](https://user-images.githubusercontent.com/25332081/205201399-2397d12a-739a-4f50-9d49-dca953ae9778.png)

**Comment by kiligir** [comment]
Came to leave the same conflict.

```
[W] libWrapper: Potential conflict detected between module Drag Ruler and module Inactive Tokens.
Module drag-ruler did not chain the wrapper for 'Token.prototype._onDragLeftDrop'. libWrapper-notifications.js:89:5.
```

**Comment by manuelVo** [comment]
Are you experiencing any actual conflict between the modules?

**Comment by manuelVo** [comment]
Sorry for the late response. Are you still experiencing that issue? If so, does the issue go away if you disable either Drag Ruler or Inactive Tokens?

**Comment by txm3278** [comment]
Sorry for the very late response, I forgot about this until recently testing v11. I don't notice any actual conflict between the modules. The warning does pop up as an overlay on screen every time I move a token for the first time on load so it would be nice if there was a way to fix this but not urgent at all.

**Comment by manuelVo** [comment]
Libwrapper is a bit overzealous when showing warnings to users. Getting such a warning is no indication that there are two modules that mod the same portion of foundry (dragging tokens in this case). In that case it shows a warning that there may or may not be a conflict (which is the warning you're getting). You can disable those warnings in the libwrapper settings by disabling "Notify GM of issues". However this has the downside that if you disable that checkbox you won't be notified of actual issues anymore either.
